rules/clean-git-history/rule.md

```diff
@@ -67,7 +67,11 @@ It's important when combining git repositories that you bring all the history wi
 :::good
 Figure: Good example
 :::
+## Descriptive commit messages
 
+It's important to have descriptive names for commits so that you can easily keep track of what was achieved after each commit was applied. Knowing what was achieved when the commits were made will make it easier to retroactively squash related commits as you'll know what work was done. In addition having descriptive commit messages makes it easier for a reviewer to see what you were trying to achieve in your pull request. 
+
+For more information about writing descriptive commit messages see this [rule](https://www.ssw.com.au/rules/use-emojis-in-your-commits/).
```

Comment:
Hey @Calinator444 

- [ ] 1. Can you read https://www.ssw.com.au/rules/descriptive-links/ and follow in the link here
- [ ] 2. Please use the relative path for the rule... (don't need "https://www.ssw.com.au/rules")

Reply:
I would change to:

> Read more about [writing descriptive commit messages](/use-emojis-in-your-commits/).

Reply:
Hi @tiagov8 
>  1. Can you read https://www.ssw.com.au/rules/descriptive-links/ and follow in the link here

**Done** - I've read the rule and I'll format my links for better SEO in the future. Thanks for the tip. 

> 2. Please use the relative path for the rule... (don't need "https://www.ssw.com.au/rules")
 
**Done** - I've removed "https://www.ssw.com.au/rules" from the link

> I would change to:
Read more about [writing descriptive commit messages](https://github.com/use-emojis-in-your-commits/).

**Done** - I've updated the link at the end of the paragraph to use that format.